support/reftest/data.yml

```diff
@@ -77,3 +77,7 @@ test_data:
 - path: /content/dist/rhel8/8.2/x86_64/baseos/iso/rhel-8.2-x86_64-boot.iso
 - path: /content/dist/rhel8/8.2/x86_64/baseos/iso/does-not-exist.iso
   state: absent
+
+# origin content failover
+- path: /_exodus-replica-dummy-object.txt
```

Comment:
Note it's also possible to add the checksum in here which would enable some more tests.  Up to you.

Reply:
Good idea, this file won't change often.